brave/brave-core
# Add link to autocomplete option in Leo settings page

Some users expect an option to en/disable Leo autocomplete suggestion to be in Leo settings. But we have all autocomplete settings in toolbar setting page. So we add a link to the page in Leo settings page.

### Demo
https://github.com/brave/brave-core/assets/5474642/9d24bf84-15bc-4859-bb59-491c5bd0c964




Resolves https://github.com/brave/brave-browser/issues/33443



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Go to brave://settings/leo
* Click "Adjust autocomplete suggestion"
* The autocomplete section in the appearance page should open
